source/docs/articles/drupal/8.md

```diff
@@ -0,0 +1,41 @@
+---
+use: [8]
+layout: default
+title: Drupal 8 on Pantheon
+---
+
+<div class="container article">
+  <div class="col-md-9">
+      <h1>Drupal 8 on Pantheon</h1>
+      <p><a href="https://www.drupal.org/drupal-8.0"> Drupal 8 RC1</a> is now available on Pantheon! You can launch Drupal 8 sites on Pantheon, but things may break and we cannot offer support at this time.  
+
+      If you want to request a new doc or have an improvement to suggest, check out the <a href="https://github.com/pantheon-systems/documentation/issues?q=is%3Aopen+is%3Aissue+milestone%3AD8">
+      issue queue</a> to see what we're planning and working on. Pull requests are always welcome.
+
+      <!-- Contrib dropdown button CTA Contribute to Docs -->
+      <div class="dropdown" id="contrib-cta-btn">
+        <button class="btn-secondary dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown"  aria-expanded="true"><span class="icon-github"></span> Contribute to Docs<span class="caret"></span></button>
+            <div class="dropdown-menu edit" aria-labelledby="dropdownMenu1">
+                <a href="https://github.com/pantheon-systems/documentation/edit/master/source{{ page.url }}.md" target="blank"><span class="glyphicon glyphicon-pencil"></span> Edit This Doc</a><br>
+                <a href="https://github.com/pantheon-systems/documentation/issues/new?title={{page.title}}%20Doc%20Update%20&body=Re%3A%20%5B{{page.title}}%5D(https%3A%2F%2Fpantheon.io{{page.url}})%0A%23%23%20Issue%20Description%3A%0A%0A%23%23%20Suggested%20Resolution%20&labels=type%3A%20doc%20update" target="blank"><span class="glyphicon glyphicon-tag"></span> Report Doc Issue</a><br>
+                <a href="https://github.com/pantheon-systems/documentation/issues/new?title=New%20Doc%20Proposal%20&body=%23%23%20Title%0A%0A%0A%23%23%20Description%0A%0A%0A%23%23%20Outline%0A%0A%0A%23%23%20Expected%20Audience%0A%0A%0A%23%23%20Path%0A(e.g.%20%60source%2Fdocs%2Farticles%2Fsites%2Fcode%60%20or%20%60source%2Fdocs%2Farticles%2Fwordpress%60)&labels=type%3A%20new%20article" target="blank"><span class="glyphicon glyphicon-file"></span> Suggest New Doc</a>
+            </div>
+        </div>
+        <h2>Drupal 8 Docs</h2>
+        {% for article in data.8 %}
+            <article>
+                <hr>
+
+                <h4><a href="{{ article.url }}">{{ article.title }}</a></h4>
+
+                <p> {{ article.description }} </p>
+            </article>
+        {% endfor %}
+        <hr>
+        <h2>Additional Resources</h2>
+
+        <h4><a href="/docs/articles/drupal/launch-check-drupal-performance-and-configuration-analysis#frequently-asked-questions">Launch Check - Drupal Performance and Configuration Analysis</a></h4>
```

Comment:
I'm not sure why we'd link to this here. Consider replacing with a ul stating what is not supported: site_audit, solr, redis